dtls.c

```diff
@@ -541,50 +541,62 @@ static uint8 compression_methods[] = {
   TLS_COMPRESSION_NULL
 };
 
+typedef enum {
+  NONE,
```

Comment:
I wonder if we should prefix these with, e.g. `DTLS_CIPHER_` or similar. Especially `NONE` might be prone to name clashes.

Reply:
What's about `KEY_EXCHANGE_...`?

Reply:
WFM